src/main/java/org/gridsuite/filter/server/dto/expertfilter/expertrule/StringExpertRule.java

```diff
@@ -31,6 +36,19 @@ public class StringExpertRule extends AbstractExpertRule {
     @Schema(description = "Value")
     private String value;
 
+    @Schema(description = "Values")
+    @JsonDeserialize(as = HashSet.class)
+    private Set<String> values;
+
+    @Override
+    public String getStringValue() {
+        if (isMultipleCriteriaOperator(this.getOperator())) { // multiple values
+            return String.join(",", this.getValues());
+        } else { // single value or absence
```

Comment:
Same here, throws a NPE if this.getValues() is null.

Reply:
Idem! Should manage at DTO level by a validator to do a cross validation between two fields. This will be corrected in another PR